bcdaworker/main_test.go

```diff
@@ -104,8 +104,6 @@ func TestWriteEOBDataToFile(t *testing.T) {
 			var jsonOBJ map[string]interface{}
 			err := json.Unmarshal(scanner.Bytes(), &jsonOBJ)
 			assert.Nil(t, err)
-			assert.NotNil(t, jsonOBJ["fullUrl"], "JSON should contain a value for `fullUrl`.")
-			assert.NotNil(t, jsonOBJ["resource"], "JSON should contain a value for `resource`.")
```

Comment:
Are there other fields we could check for? Based on https://www.hl7.org/fhir/explanationofbenefit.html, there are some that should always be there, like `status` and `type`.

Reply:
yeah I was looking for something to key off of, but I would like to expand this check to our Coverage and Patient endpoints as well. Again tho, this file only tests our EOB stuff and not the others but we've already identified that in a future ticket. Will only implement this for EOB at the moment as suggested.